reference/services.html.md.erb

```diff
@@ -57,3 +57,111 @@ If you don't specify handlers, we just forward TCP to your application as is. Th
 
 The `proxy_proto` handler adds information about the original connection, including client IP + port and server IP + port (from the client's perspective).  Most applications need additional logic to accept the proxy protocol, either using a prebuilt library or implementing the [proxy protocol](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt) directly.
 
+
+## _HTTP Options_
+
+`http_options` is used to specify additional configuration for HTTP handlers. 
+
+```toml
+[[services.ports]]
+handlers = ["http"]
+port = "80"
+
+  [services.ports.http_options]
+  compress = true
+
+    [services.ports.http_options.response]
+    header = { myheader = "fly", via = false}
+
+    [services.ports.http_options.https_redirect]
+    enable = true
+    port = 777
+    status_code = 308
+```
+
+### Compress
+
+By default, Fly will compress response bodies. You can switch it off by setting the value to `false`.
+
+### Response
```

Comment:
Same here, maybe `Modifying Response Headers`. And below, "The `response` section allows you to add or remove HTTP headers to the response.